js/about/newtab.js

```diff
@@ -257,6 +261,79 @@ class NewTabPage extends React.Component {
   }
 
+  launchBraveCore () {
+    const braveCoreInstallPath = this.state.versionInformation && this.state.versionInformation.getIn(['initState', 'braveCoreInstallPath'])
+    if (braveCoreInstallPath) {
+      aboutActions.launchBraveCore()
+    }
+  }
+
+  openHelp () {
+    window.location = 'https://support.brave.com/hc/en-us/articles/360018538092'
+  }
+
+  getDeprecatedText () {
+    const muonVersion = this.state.versionInformation && this.state.versionInformation.get('browserLaptop')
+    const formattedMuonVersion = muonVersion
+      ? ('(' + muonVersion + ')')
+      : ''
+    const braveCoreInstalled = (this.state.versionInformation && this.state.versionInformation.getIn(['initState', 'braveCoreInstalled'])) || false
+    const braveCoreVersion = braveCoreInstalled && this.state.versionInformation && this.state.versionInformation.getIn(['initState', 'braveCoreVersion'])
+    const formattedBraveCoreVersion = braveCoreVersion
+      ? ('(' + braveCoreVersion + ')')
+      : ''
+    let braveCoreFriendlyVersion = (braveCoreVersion && braveCoreVersion.split('.').length === 3)
+      ? braveCoreVersion.split('.').slice(0, 2).join('.')
+      : undefined
+    const launchButtonText = braveCoreFriendlyVersion
+      ? `Launch Brave ${braveCoreFriendlyVersion}`
+      : 'Launch Brave'
+
+    if (braveCoreInstalled) {
+      return <div className='deprecationNotice'>
+        <div>
+          <span className='note'>Note:</span>&nbsp;
+          A newer version of Brave {formattedBraveCoreVersion} has already been installed.
+          This version of Brave {formattedMuonVersion} is no longer supported and will not be updated.
+        </div>
+        <div style={{marginTop: '20px'}}>
+          To avoid potential security risks, please move over to the latest version of the Brave Browser.
+        </div>
+        <div style={{marginTop: '40px'}}>
+          <span style={{width: '50%', textAlign: 'center', display: 'inline-block'}}>
+            <a href='https://support.brave.com/hc/en-us/articles/360018538092'>Learn more…</a>
+          </span>
+          <BrowserButton
+            primaryColor
+            l10nId={launchButtonText}
+            inlineStyles={{width: '50%'}}
+            onClick={this.launchBraveCore.bind(this)}
+          />
+        </div>
+      </div>
+    }
+
+    return <div className='deprecationNotice'>
+      <div>
+        <span className='note'>Hello!</span> This version of Brave {formattedMuonVersion} is no longer supported and will not be updated.
+        {
+          isLinux()
+            ? <span>&nbsp;To avoid potential security risks, please follow these <a href='https://brave-browser.readthedocs.io/en/latest/installing-brave.html#linux'>instructions</a> to upgrade to the latest version of the Brave Browser.</span>
+            : <span>&nbsp;To avoid potential security risks, please <a href='https://brave.com/download'>download the latest version</a> of the Brave Browser.</span>
```

Comment:
do we need to localize this?

Reply:
I considered that- but since it's unlikely we'll get translations (we'd need to wait for them to be done in order to package them), I didn't translate it

cc: @bbondy @rebron

Reply:
seems reasonable